scripts/generate_mkdocs.py

```diff
@@ -161,7 +161,7 @@ def main() -> None:
     # Format rules docs
     add_no_escape_text_plugin()
     for rule_doc in Path("docs/rules").glob("*.md"):
-        mdformat.file(rule_doc, extensions=["mkdocs", "admonition", "no-escape-text"])
+        mdformat.file(rule_doc, extensions=["mkdocs", "admon", "no-escape-text"])
```

Comment:
This was a breaking change in `mdformat-admon`, in the 2.0.3 release: https://github.com/KyleKing/mdformat-mkdocs/commit/6a329979e27da47a215607d4e30f61d22e683b89